### DISCUSSION TITLE
DISABLE_AUTO_TITLE="true", how to revert to default zsh title?

### DISCUSSION THREAD

**Discussion by willenglishiv** [ohmyzsh/ohmyzsh#12759] [Help] [answered]
I just want my terminal title to go back to what it used to be before I installed omz.

which I believe is user@host: $PWD

If I use DISABLE_AUTO_TITLE="true" then nothing is set.  I see some code in bashrc but I don't understand it 100%.

**Comment by carlosala** [comment, marked as answer]
Hi! You should write `DISABLE_AUTO_TITLE=true` in your `.zshrc`, before the line `source oh-my-zsh.sh`.
I hope this guides you in the right way!
Thanks!

**Comment by willenglishiv** [comment]
Hi, 

Maybe I wasn't clear. I did that, but when I do, nothing is set.

![image](https://github.com/user-attachments/assets/864f3db6-0fd2-42de-aadf-8cf759044aa9)

![image](https://github.com/user-attachments/assets/08891938-ad5d-4d79-94db-cb528efc4e3b)

how do I get it back to before I installed omz?  This example is from WSL / Ubuntu but I could fire up a virtual machine and it will be the same behavior.
